### PR TITLE
Plans: add plan-feature-items

### PR DESCRIPTION
**Pull request by gwwar**
This is part of the plans redesign, which breaks out the plan features item component from @adambbecker's work in #5715
b6a11926-2cd1-11e6-8322-f88c322fbdcd

This PR adds the plans feature items to [devdocs](http://calypso.localhost:3000/devdocs/app-components/plan-features)
screen shot 2016-06-09 at 5 14 25 pm

## Testing Instructions
- No functional changes to Calypso.
- Navigate to http://calypso.localhost:3000/devdocs/app-components/plan
- You should see the PlanFeatures component as shown above

cc @lamosty @rralian @mtias @rodrigoi

### CODE REVIEW

File: client/my-sites/plan-features/list.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+	render() {
+		return (
+			<ul className="plan-features__item-list">
+				{ this.props.children }
+			</ul>
+		);
+	}
+}
+
+export default PlanFeaturesItemList;
```
Comment (lamosty) [inline client/my-sites/plan-features/list.jsx]: Could we make this component (and the one above) stateless? :) Something like this:

```js
const PlanFeaturesItemList = ( { children } ) => {
	return (
		<ul className="plan-features__item-list">
			{ children }
		</ul>
	);
};

export default PlanFeaturesItemList;
```

**Comment by lamosty** [comment]
Nice work @gwwar! Components render fine.

I wonder, is there a way to make the plans in devdocs align left to each other? So it would look like on `/plans`. I'm going to research this a bit.

**Comment by mtias** [comment]
> I wonder, is there a way to make the plans in devdocs align left to each other? So it would look like on /plans. I'm going to research this a bit.

Shouldn't be hard just wrapping them in a div with a style attribute for flex properties.

**Comment by lamosty** [comment]
> Shouldn't be hard just wrapping them in a div with a style attribute for flex properties.

Right. But we still don't have `PlanFeatures` component so first, we would have to wrap everything in `PlanFeaturesList` with flex props and then wrap each plan in `PlanFeatures` with more props. I can try it.

**Comment by gwwar** [comment]
> I wonder, is there a way to make the plans in devdocs align left to each other? 

The current state in devdocs is temporary for now. As you might expect, I was planning to eventually show a single `PlanFeatures` component followed by a `PlanFeaturesList` when they become available

File: client/my-sites/plan-features/list.jsx (hunk 1 above)
Comment (gwwar) [inline client/my-sites/plan-features/list.jsx]: Just remember that we won't be able to reference stateless components. It's only an issue if we need a popover.

Comment (lamosty) [inline client/my-sites/plan-features/list.jsx]: Thanks, I was not aware of that.

Comment (gwwar) [inline client/my-sites/plan-features/list.jsx]: Updated in beccea5

**Comment by lamosty** [comment]
LGTM :ship:

**Comment by gwwar** [comment]
Thanks for the reviews!

**Pull request merged**